sssd_test_framework/utils/authentication.py

```diff
@@ -341,23 +351,32 @@ def password_expired(self, username: str, password: str, new_password: str) -> b
         return result.rc == 0
 
     def passkey_with_output(
-        self, username: str, *, pin: str | int, device: str, ioctl: str, script: str, command: str = "exit 0"
+        self, username: str, *, device: str, ioctl: str, script: str, pin: str | int | None = None,
+            interactive_prompt: str = "Insert your passkey device, then press ENTER.", 
+            touch_prompt: str = "Please touch the device.", command: str = "exit 0",
```

Comment:
Do we assume here that the prompts will not be changed from the default via options below?
~~~
           interactive_prompt
               to change the message of the interactive prompt.

           touch_prompt
               to change the message of the touch prompt.
~~~

Reply:
We can change it, but set default values also,
https://github.com/SSSD/sssd/pull/7611/files#diff-d6f662469d57a0e4deb526c1d7e86dd9402ebd45116869053828053506d0007aR798

this test check about the updated prompt options from the sssd.conf.